packages/snap-tracker/src/Tracker.ts

```diff
@@ -475,10 +490,10 @@ export class Tracker {
 			},
 		},
 		order: {
-			transaction: (data: OrderTransactionData, siteId?: string): BeaconEvent | undefined => {
+			transaction: (data: OrderTransactionData, siteId?: string, currency?: string): BeaconEvent | undefined => {
```

Comment:
I think we can remove the currency param here and instead assume that the currency has already been set on the context.